spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/LoadBalancerProperties.java

```diff
@@ -572,4 +585,22 @@ public void setIncludePath(boolean includePath) {
 
 	}
 
+	public static class InterfaceClients {
```

Comment:
Seems a bit strange for the load balancer to know about interface clients

Reply:
Removed